sdk/digitaltwins/Azure.DigitalTwins.Core/tests/prerequisites/prerequisite readme.md

```diff
@@ -13,13 +13,19 @@
   - If it isn't, update it
 - Use this link to install [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest])
 
-## Delete
+### Install Bicep
 
-To delete the digital twins instance, you need to first delete the endpoint added by the script (the service doesn't yet support cascading delete).
+- Install using the instructions in [bicep](https://github.com/Azure/bicep/blob/main/docs/installing.md)
 
-1. To do this, run the command `az dt endpoint delete -n <dt name> -g <rg name> --en someEventHubEndpoint`.
-1. If you have other endpoints that have been added outside this script, you can discover them with the command `az dt endpoint list -n <dt name> -g <rg name>`.
-1. Then delete them with the same command in step 1.
+### Run the setup script - The script outputs [test-resources.bicep](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/digitaltwins/test-resources.bicep) file in the root folder [digitaltwins](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/digitaltwins)
```

Comment:
Too many words in a header

Reply:
It felt concise in the rich text view. Let me know if you think this looks too long.
![image](https://user-images.githubusercontent.com/14096264/98162453-3949d600-1e96-11eb-9d30-44b86fabfad4.png)

Reply:
Headers should be brief, not full on sentences. So either make it normal body text, or give the header a brief name, and put the rest of this as body.